recipes/libcudss/meta.yaml

```
@@ -0,0 +1,155 @@
+
+{% set version = "0.3.0.9" %}
+{% set lib_version = version.split(".")[0]+"."+version.split(".")[1]+"."+version.split(".")[2] %}
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "placeholder" %}  # [osx]
+{% set extension = "tar.xz" %}  # [unix]
+{% set platform = "windows-x86_64" %}  # [win64]
+{% set extension = "zip" %}  # [win64]
+
+{% set cuda_compiler_version = cuda_compiler_version|default("None") %}
+{% set cuda_major = environ.get("cuda_compiler_version", "12.0").split(".")[0] %}
+
+package:
+  name: libcudss-split
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cudss/redist/libcudss/{{ platform }}/libcudss-{{ platform }}-{{ version }}_cuda{{ cuda_major }}-archive.{{ extension }}
+  sha256: 6eadd70e6442b43194e697c13080dae3cf1771257a334e7d7b658825a3e73a3b  # [aarch64 and (cuda_compiler_version or "").startswith("12")]
+  sha256: acc90b40334cd2e4409ec9e4200c8276f7acd50d61b7349b4ffdaeda4b2bf515  # [linux64 and (cuda_compiler_version or "").startswith("12")]
+  sha256: ac0979610d3eda9e6ce803380587118c6ec72b7dc580e6e96392ac065010196f  # [win64   and (cuda_compiler_version or "").startswith("12")]
+
+build:
+  number: 0
+  skip: true  # [cuda_compiler_version in (None, "None", "11.8")]
+  skip: true  # [not (linux64 or aarch64 or win64)]
+  script:
+    - cp -rv include $PREFIX  # [linux]
+    - cp -rv lib $PREFIX  # [linux]
+    - cp -rv include %PREFIX%  # [win]
+    - cp -rv lib %PREFIX%  # [win]
+    - cp -rv bin %PREFIX%  # [win]
```

For Windows we've been using the `LIBRARY_LIB` and `LIBRARY_BIN` locations rather than `PREFIX`
```suggestion
    - cp -rv lib %LIBRARY_LIB%  # [win]
    - cp -rv bin %LIBRARY_BIN%  # [win]
```